views.py

```diff
@@ -154,6 +154,39 @@ def page_detail(request, page_id, book_id=None):
         curr_annot = get_annotation_detail(annotation)
         context['annotations'].append(curr_annot)
 
+    # Previous/next page queries
+    pagenum = int(page_json['rel_has_pagination_ssim'][0])
+    base_url = "https://%s/api/pub/search/?q=ir_collection_id:621+AND+rel_is_part_of_ssim:\"%s\"+AND+%s+AND+display:BDR_PUBLIC&fl=pid,rel_has_pagination_ssim,rel_is_part_of_ssim"
+    full_url = base_url % (BDR_SERVER, book_pid, "(rel_has_pagination_ssim:%d+OR+rel_has_pagination_ssim:%d)" % (pagenum - 1, pagenum + 1))
+
+    r = requests.get(full_url, timeout=60)
```

Comment:
It looks like there's a book_json variable earlier in this function. That should contain a 'relations' section, which has a 'hasPart' section (eg. see https://repository.library.brown.edu/api/pub/items/bdr%3A253364/). That 'hasPart' section lists all the pages of the book (in order), so you should be able to grab the prev/next pids from that list. That way, we'd be able to eliminate one call to the api, which should speed up the page load a bit.